src/groups/special_euclidean.jl

```diff
@@ -721,7 +670,7 @@ function project(M::SpecialEuclideanInGeneralLinear, p, X)
     G = M.manifold
     np, hp = submanifold_components(G, p)
     nX, hX = submanifold_components(G, X)
-    return ArrayPartition(hp * nX, hX)
+    return ArrayPartition(nX, hX)
```

Comment:
Does that mean we changed representation of tangent vectors? Just curious, whether this is breaking.

Reply:
Yes, the representation is now left invariant instead.

Reply:
Ok, but this is a breaking change isn't it?

Reply:
To be clear, I do not mind doing a breaking release, just that _if_ we only do them very carefully in order to not do them too often; until now we had 8 breaking releases in a bit more than 5 years, recently more like once a year or less. 

If this is breaking we should definetly remark that in the News.md what we broke and how to change existing code to work the same as before. That's the reason for the question.

Reply:
I'm going to make some additional work myself so that the old representation of SE(n) can still be used but there still likely will be something breaking.

Reply:
If you don't want breaking change, just rename the current `SpecialEuclidean` to something else, like `Displacement`, and keep the old, incomplete implementation of `SpecialEuclidean`.

Reply:
No, we have a few things we could do for a breaking change, just that we consider that carefully. Here, that the tangent vectors are then unified for the Lie groups is a good reason – my main point is then to carefully document (in the News.md) what a user has to change in their code.

Reply:
If you have a **stored** tangent vector `X` at point `p` of `G = SpecialEuclidean(n)`, you would have to replace it with `translate_diff(G, inv(G,p), p, X)`. **Unfortunately** this method (`translate_diff`) is not available in `SpecialEuclidean` (because the current implementation of `SpecialEuclidean` is incomplete).

Other than that, (if you do not *store*, or manually manipulate, any tangent vectors), the transition should be transparent.

Reply:
I would just add a new type parameter for `SpecialEuclidean` that determines the storage. AFAIK JuliaRobotics currently relies on our storage and I wouldn't like to force them to adapt everything.

Reply:
One idea could be to use some `TangentvectorRepresentation` and `LIeAlgebraRepresentation` for that distinction? Do we have a `TVectorRepresentation` supertype? If not we could either introduce that or use subtypes of `TVector` (though that might be misleading).

Reply:
So far we've only differentiated representations by custom types, so it will be a new thing. I think we could have `TVectorRepresentation` abstract type for that purpose.